packages/backend/src/app/instance/instance.controller.ts

```diff
@@ -50,4 +50,17 @@ export const instanceController = async (app: FastifyInstance) => {
         });
         reply.status(StatusCodes.OK).send();
     });
+    // update status
+    app.post(
+        "/updateStatus",
```

Comment:
The naming convention for url is update-status, but need to think of the naming.